__tests__/articles.test.js

```diff
@@ -18,7 +18,163 @@ afterAll(() => {
+            body: "Well? Think about it.",
+            created_at: "2020-06-06T09:10:00.000Z",
+            votes: 0,
+            article_img_url:
+              "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+          },
+          {
+            article_id: 10,
+            title: "Seven inspirational thought leaders from Manchester UK",
+            topic: "mitch",
+            author: "rogersop",
+            body: "Who are we kidding, there is only one, and it's Mitch!",
+            created_at: "2020-05-14T04:15:00.000Z",
+            votes: 0,
+            article_img_url:
+              "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+          },
+          {
+            article_id: 4,
+            title: "Student SUES Mitch!",
+            topic: "mitch",
+            author: "rogersop",
+            body: "We all love Mitch and his wonderful, unique typing style. However, the volume of his typing has ALLEGEDLY burst another students eardrums, and they are now suing for damages",
+            created_at: "2020-05-06T01:14:00.000Z",
+            votes: 0,
+            article_img_url:
+              "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+          },
+          {
+            article_id: 8,
+            title: "Does Mitch predate civilisation?",
+            topic: "mitch",
+            author: "icellusedkars",
+            body: "Archaeologists have uncovered a gigantic statue from the dawn of humanity, and it has an uncanny resemblance to Mitch. Surely I am not the only person who can see this?!",
+            created_at: "2020-04-17T01:08:00.000Z",
+            votes: 0,
+            article_img_url:
+              "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+          },
+          {
+            article_id: 11,
+            title: "Am I a cat?",
+            topic: "mitch",
+            author: "icellusedkars",
+            body: "Having run out of ideas for articles, I am staring at the wall blankly, like a cat. Does this make me a cat?",
+            created_at: "2020-01-15T22:21:00.000Z",
+            votes: 0,
+            article_img_url:
+              "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+          },
+          {
+            article_id: 7,
+            title: "Z",
+            topic: "mitch",
+            author: "icellusedkars",
+            body: "I was hungry.",
+            created_at: "2020-01-07T14:08:00.000Z",
+            votes: 0,
+            article_img_url:
+              "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+          },
+        ];
+        expect(body.articles).toEqual(expected);
+      });
+  });
+});
+
```

Comment:
if you use something like `jest-sorted` you won't need to hardcode so much data here to check 👀 this will check the data you're already seeding the database with is coming back in created_at order correctly. You also need to include the comment_count too.

Reply:
hi Adam, In the test could you please test for the presence of the other properties please?